libr/asm/arch/gb/gbasm.c

```diff
@@ -123,17 +123,24 @@ static bool gb_parse_ld1 (ut8 *buf, const int minlen, char *buf_asm) {
 			return false;
 		}
 		buf[0] = (ut8)(0x40 + (i * 8));
-		if ((i = gb_reg_idx (buf_asm[5])) == (-1)) {
-			return false;
+		i = gb_reg_idx (buf_asm[5]);
+		if (i == (-1)) {
+			if (strncmp(&buf_asm[5], "[hl]", 4)) {
```

Comment:
buf_asm + 5, and you can use r_str_ncasecmp() to support uppercase register namings, (and add missing space before (

Reply:
uppercase is supported (i tested it), everything gets converted to lowercase befor